src/app/Fake.IO.FileSystem/Templates.fs

```diff
@@ -0,0 +1,26 @@
+/// NOTE: Maybe this should be an extra module?
+/// Contains basic templating functions. Used in other helpers.
+module Fake.IO.FileSystem.Templates
```

Comment:
Yes, currently this is quite coupled throughout other modules. I don't like this module ... Kind of broken by design.

Reply:
But works in practice /shrug